.github/workflows/smoke_test.yml

```diff
@@ -0,0 +1,60 @@
+name: AWS SDK Smoke Tests
+on:
+  push:
+    branches:
+      - main
+      - stable
+      - next
+    paths:
+      - 'packages/aws_sdk/**/*.dart'
+      - 'packages/smithy/**/*.dart'
+  pull_request:
+    paths:
+      - 'packages/aws_sdk/**/*.dart'
+      - 'packages/smithy/**/*.dart'
+  schedule:
```

Comment:
nit: could this have a little comment saying when this runs for ppl (like me) who have to google cron syntax for this?